src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -316,6 +358,22 @@ export const PropertyList = ({
                     'data-testid': 'removeOverflow'
                 }
             ];
+
+            // If item is the first one, remove the move up option
+            if (index === 0) {
+                menuItems = menuItems.filter(
+                    (menuItem) => menuItem.key !== 'moveUp'
```

Comment:
If we're using this key in multiple places, let's define it as a constant at the top of the file.